scripts/manage/landing_page_base.md

```diff
@@ -179,6 +179,32 @@ Update the commands and documentation in your management environment to the late
 ~/spinnaker-for-gcp/scripts/manage/connect_to_redis.sh
 ```
 
+### Restore a backup to Cloudshell
+
+Restore a backup of the halyard configuration and deployment configuration from Cloud Source Repositories to your Cloudshell. 
+
+```bash
+~/spinnaker-for-gcp/scripts/manage/restore_backup_to_cloudshell.sh -p $PROJECT_ID -r $CONFIG_CSR_REPO -h GIT_HASH
+```
+
+To apply the halyard config changes to the cluster:
```

Comment:
Looks like `push_and_apply` and `setup` are both referenced when the `restore_backup_to_cloudshell` scripts completes. Do we need the instructions in both locations?

I'd lean towards dropping them from the markdown unless you feel strongly about having them in both locations.

Reply:
Agreed, I will remove it from the markdown.